client-src/default/index.js

```diff
@@ -220,21 +220,30 @@ if (
 ) {
   protocol = self.location.protocol;
 }
+
+// default values of the sock url if they are not provided
+let sockHost = hostname;
+let sockPath = '/sockjs-node';
+let sockPort = urlParts.port;
+if (urlParts.path != null && urlParts.path !== '/') {
```

Comment:
Please use strict equality operator. Or are you intentionally doing so to include `undefined`?

Reply:
That's how it was here: https://github.com/webpack/webpack-dev-server/blob/master/client-src/default/index.js#L235. I kept it that way under the assumption that it might need to catch `undefined`.

Reply:
I see. I didn't know that it already used...

Reply:
@Loonride i think we should fix using `!= (`==` and etc) in our code base

Reply:
@evilebottnawi should I fix it here?

Reply:
I want to change `!==`, but its change is the breaking change, so please rewrite as below.

`if ((urlParts.path !== null || urlParts.path !== undefined) && urlParts.path !== '/') {`

Reply:
I think `urlParts.path != null` would turn into `(urlParts.path !== null && urlParts.path !== undefined)`

Reply:
You're right. My mistake.